### ISSUE TITLE
How do you add custom deck images?

### ISSUE THREAD

**Issue by Madmaxneo** [alexkurowski/solo-toolkit#7]
I searched and could find no folder named "Decks". 
I created a folder named Decks and added two folders with cards in them, but the only thing in the card options are "Tarot" and "Standard". I did a search throughout my vault for names like "Decks", "Tarot", and "Standard" but nothing comes up pertaining to this plugin. 

Where do I add the custom deck images?

**Comment by alexkurowski** [comment]
Yeah, you have to create the folder yourself in your vault, and it must be at the root of your vault by default. If you'd like to nest your decks somewhere else, you can change the folder in plugin settings (e.g. set the option to `Assets/Decks` or something).

Also worthy of note, your folder needs to be named `Decks` and not `decks`, but I think you're good there.

You also need to hit the refresh button, or switch to another tab and back, for changes to take place 😓

**Comment by TCArknight** [comment]
After watching Josh P's latest video, it also appears that the images of the cards in your Decks folder must be PNG format. JPG and WEBP display blanks when drawn.

**Comment by alexkurowski** [comment]
JPG format should work correctly if you're on version 0.3.6 or later

There is a PR for WEBP support https://github.com/alexkurowski/solo-toolkit/pull/8

**Comment by Madmaxneo** [comment]
> After watching Josh P's latest video, it also appears that the images of the cards in your Decks folder must be PNG format. JPG and WEBP display blanks when drawn.

Not true, at least in the versions I downloaded. I converted a bunch of cards from gif to jpg and they work just fine in the program.

**Comment by alexkurowski** [comment]
There indeed was a bug in prior versions where files with extension ".jpeg" wouldn't load, but ".jpg" worked fine. So that might be the reason.
Also, I should add support for gifs! Would be fun to have animated cards.

**Comment by Madmaxneo** [comment]
> There indeed was a bug in prior versions where files with extension ".jpeg" wouldn't load, but ".jpg" worked fine. So that might be the reason. Also, I should add support for gifs! Would be fun to have animated cards.

...and disregard all that work I did converting the gifs to jpg files? Oh, the nerve! J/k 
It would be awesome if you could support for gif files!

Hopefully you'll fix the issue with it creating a new image whenever you drag a card to canvas or excalidraw. 
I'm happy either way.
